browser/greaselion/greaselion_tab_helper.h

```diff
@@ -0,0 +1,46 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_GREASELION_GREASELION_TAB_HELPER_H_
+#define BRAVE_BROWSER_GREASELION_GREASELION_TAB_HELPER_H_
+
+#include <map>
```

Comment:
not sure if any of these are needed. I normally just let lint tell me which ones are missing